bitnami/thanos/values.yaml

```diff
@@ -2582,6 +2582,567 @@ metrics:
     ##
     # scrapeTimeout: 10s
 
+  ## PrometheusRule CRD configuration
+  ##
+  prometheusRule:
+    enabled: false
+    ## Namespace in which the PrometheusRule CRD is created
+    ##
+    namespace: monitoring
+    ## List of rules for each component
+    ##
+    rules:
```

Comment:
This is something that I noticed taking a look other charts. We are not including the rules by default, ex:

https://github.com/bitnami/charts/blob/77df34ca6871e06c7bcaa51291b90afe964ec323/bitnami/redis/values.yaml#L1259

Please, could you comment them? I mean, this is something that the users must to modify to configure them. So, I suggest to comment them. Thanks again, you are really close

Reply:
Indeed they are not included by default!

Reply:
sorry @francoisminaud, I used a bad word (English is not my mother language). If you take a look in other charts, we try to include new features trying to avoid specific configurations (sometimes we can do it and sometimes not). If you take a look for example Redis or Mariadb-galera, you can see that we are including the prometheusRules feature but we are not setting up any rules. We don't want to establish specific rules for the users. You did a really hard work and we don't want to lose it, for that I decided to move your rules to the documentations repo. Please, remove the rules to keep following the same way that we did in other charts. 

Thanks a lot again for your really hard work and again sorry.